lib/helpers/metaLoader.js

```diff
@@ -70,6 +70,30 @@ module.exports = class MetaLoader {
       });
   }
 
+  async getLookupFieldsModelWithTypeOfSearch(typeOfSearch) {
+    if (typeOfSearch === 'uniqueFields')
+    return await this.connection.describe(this.configuration.sobject)
+      .then(async meta => {
+        const model = {};
+        await meta.fields
+          .filter(field => field.type === 'id' || field.unique)
+          .forEach(field => {
+            model[field.name] = field.label;
+          });
+        return model;
+      });
+    else
+      return await this.connection.describe(this.configuration.sobject)
+          .then(async meta => {
+            const model = {};
+            await meta.fields
+                .forEach(field => {
+                  model[field.name] = field.label;
```

Comment:
I wonder if it would be more beneficial to do 
```
model[field.name] = `${field.label} (${field.name})`;
```

Reply:
done